ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh

```diff
@@ -44,6 +44,15 @@ data:
 EOF
 fi
 
+
+# Alibaba requires PVCs to be 20GB
+STORAGE="20Gi"
+#PLATFORM=$(/tmp/yq r "${SHARED_DIR}/install-config.yaml" '.platform' | awk -F':' '{print$1}')
+#if [[ "${PLATFORM}" == "alibabacloud" ]];
+#then
+#    STORAGE="20Gi";
```

Comment:
I don't feel to set on making this sizing conditional on the chosen platform, but I don't think we need to keep these dead comments around.  Can we either revive this approach, or drop the commented code and explain why we skipped making it platform-specific?

Reply:
@kwoodson just sharing that [doc](https://www.alibabacloud.com/help/en/doc-detail/122389.html) with the reference of minimal disk size:
> ESSD  PL1
> Capacity range (GiB): 20~32,768